### DISCUSSION TITLE
Error Magma

### DISCUSSION THREAD

**Discussion by Gonzaloejuarez** [mitre/caldera#2951] [General]
Good morning

I'm having a problem installing the Caldera software on my ubuntu server 22.04.

I need your help because I can't find the solution.

![Captura de pantalla 2024-04-17 102606](https://github.com/mitre/caldera/assets/75266150/07417c23-2b16-405e-851b-d1b38ae1c5a5)

**Comment by Guil33** [comment]
It's most likely your NodJs version that's outdated.
If you're having troubles, try to follow these steps: https://github.com/mitre/caldera/issues/2943#issuecomment-2051186848

**Reply by Gonzaloejuarez** [reply]
Thank you for your reply!

The error I showed above no longer shows it. But now this mistake is coming out when I lift Caldera. 

Go to that dashboard and then go back to the Login.

![image](https://github.com/mitre/caldera/assets/75266150/a170d298-b95b-43cb-a3ff-582eb9a0c2a5)


When I pick up Caldera I get this on the console

![image](https://github.com/mitre/caldera/assets/75266150/0c6615f3-0bd3-4320-b5ed-d0d3856658b7)

**Reply by afmurillo** [reply]
Hey Gonzalo, I managed to solve the docker error by doing
$: sudo apt install docker
$: python3 -m pip install --upgrade docker

Nevertheless, I am facing the same issue you're facing, I login with the credentials stored in local.yml, but I also get the behaviour of "Go to that dashboard and then go back to the Login."

**Reply by afmurillo** [reply]
Nevermind, following this solved my issue: https://github.com/mitre/caldera/discussions/2878
